react/tic-tac-toe/src/app/screens/Game/index.js

```diff
@@ -65,23 +72,28 @@ import Board from './components/Board';
     }
     
     return (
-      <div className={styles.game}>
-        <div className={styles.gameBoard}>
-        <Board
-          squares={current.squares}
-          onClick={(i) => this.handleClick(i)}
-        />
-        </div>
-        <div className={styles.gameInfo}>
-          <div>{ status }</div>
-          <ol>{ moves }</ol>
+      <div>
+        <button type="button" onClick={() => this.logout()}>
```

Comment:
Create logout as an arrow function and here just use the fn
`<button type="button" onClick={this.logout}>`